src/main/java/com/atlassian/db/replica/api/AuroraConnectionDetails.java

```diff
@@ -36,9 +56,17 @@ public Builder password(String password) {
             return this;
         }
 
+        /**
+         * @return new instance of this class
```

Comment:
comment "@return new instance of this class" doesn't add any value, IMO it make code more bloated

Reply:
will remove the unnecessary doc